streaming_to_playlist.py

```diff
@@ -33,42 +36,24 @@ class StreamingToPlaylist(SpotifyHandler):
     """
```

Comment:
please remove this comment:
"The scope in config.py must contain the following: "playlist-modify-private playlist-modify-public"